flink/v1.18/flink/src/main/java/org/apache/iceberg/flink/source/RowDataRewriter.java

```diff
@@ -84,7 +85,8 @@ public RowDataRewriter(
             format,
             table.properties(),
             null,
-            false);
+            false,
+            DeleteGranularity.FILE);
```

Comment:
is the default change intentional?

Reply:
We do not write delete files, so it has no impact.

Reply:
Also the goal is to used `DeleteGranularity.FILE` in Flink as a general rule